umh-core/pkg/config/manager.go

```
@@ -299,3 +358,82 @@ func (m *FileConfigManagerWithBackoff) IsPermanentFailure() bool {
 func (m *FileConfigManagerWithBackoff) GetLastError() error {
 	return m.backoffManager.GetLastError()
 }
+
+// WriteConfig delegates to the underlying FileConfigManager
+func (m *FileConfigManagerWithBackoff) writeConfig(ctx context.Context, config FullConfig) error {
```

golangci-lint (1.64.8)

func `(*FileConfigManagerWithBackoff).writeConfig` is unused

(unused)